packages/svelte-materialify/src/components/DataTable/_variables.scss

```diff
@@ -0,0 +1,7 @@
+@import '../../styles/variables';
+
+$tbl-border-color: rgba(0, 0, 0, 0.12);
+$tbl-border-radius: $border-radius-root !default;
+$tbl-border-width: 1px;
+$tbl-background-hover: rgba(0, 0, 0, 0.04);
+$tbl-head-font-weight: map-get($font-weights, 'medium');
```

Comment:
Anyone reviewing this: Could you please check this? Especially the colors and how they interact with the dark theme...
Thanks